draft-ietf-quic-transport.md

```diff
@@ -849,7 +849,9 @@ initial_stream_id_for_type) can be opened (see {{long-packet-types}}).  Initial
 limits are set in the transport parameters (see
 {{transport-parameter-definitions}}) and subsequently limits are advertised
 using MAX_STREAMS frames ({{frame-max-streams}}). Separate limits apply to
-unidirectional and bidirectional streams.
+unidirectional and bidirectional streams.  If the maximum stream ID that can
+be opened is greater than 2^62 - 1, then the receiver MUST interpret it no
+stream limit and the receiver MUST NOT close the connection.
```

Comment:
I don't think this says quite what you mean -- I can send you a MAX_STREAMS frame with a very large value and prohibit you from closing the connection thereafter?  Of course, the only way to violate that prohibition is to close the connection, and the only punishment is to close the connection, so this would be entirely toothless even if it were what you meant.

```suggestion
stream limit; this MUST NOT be treated as an error.
```